packages/lab/src/color-chooser/Color.ts

```diff
@@ -1,4 +1,4 @@
-import tinycolor from "tinycolor2";
+import tinycolor2 from "tinycolor2";
```

Comment:
It's more than cspell isn't smart enough without using the lint plugin to ignore these kinds of cases. As an alternative it could also be called tinyColor.